accessibility/accessibility-checklist.md

```diff
@@ -1,94 +1,218 @@
+
+We do this:
+
+```
+<p>This document has the lang="en" attribute, and this text matches the document language.</p>
+<p lang="jp">このテキストは日本語です</p>
+```
+
+Resources: 
+
+* [Article: Using the HTML lang attribute](https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/)
+* [Samurai errata GL4: Clarify natural-language usage](http://www.wcagsamurai.org/erratas/errata-listing/#GL4)
+
+
+### Use a logical source order
+
+Your page should still make sense when you look at it without CSS. This is a core [progressive enhancement technique](../practices/progressive-enhancement.md). 
+
+Don't surprise users by using CSS to make changes to apparent source order, e.g. by using floats and absolutely-positioned elements. 
+
+Flexbox and CSS grids are particularly vulnerable to apparent source order surprises, and you should take extra care when using them. 
+
+Resources:
+
+* [Article: Logical document structures](http://www.bettertesting.co.uk/content/?p=1619)
+* [Article: Source order matters](http://adrianroselli.com/2015/09/source-order-matters.html)
+* [Tool: Wave Extension](https://wave.webaim.org/extension/)
+
+
+### Use headings correctly
+
+The [HTML5 outline algorithm](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines) can't yet be relied on to convey information to users. Freely use HTML5 sectioning elements, but use headings to define the structure of your document. 
+
+Use an appropriate heading for each section. You can visually hide headings, but make sure they're still accessible to screen reader users. 
+
+Don't skip heading levels.
+
+Resources: 
+
+* [Article: Semantic Structure](https://webaim.org/techniques/semanticstructure/)
+* [Tool: HeadingsMap extension for Chrome](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en)
+* [Tool: HeadingsMap extension for Firefox](https://addons.mozilla.org/en-GB/firefox/addon/headingsmap/)
+
+
+### Use progressive enhancement
+
+Progressive enhancement means starting with a logical and semantic HTML document with all key functionality, then enhancing it with an aesthetic layer (CSS) and behavioural layer (JavaScript). It has benefits beyond accessibility but is a key starting point for creating accessible experiences. 
+
+Resources: 
+
+* [Playbook: Progressive enhancement](../practices/progressive-enhancement.md)
+
+
+## Media
+
+### Audio or video elements should be accessible
+
+This could involve adding [captions to videos](https://www.youtube.com/watch?v=5AXApBbj1ps), providing [transcripts of audio content](https://www.nature.com/nature/podcast/v546/n7658/nature-2017-06-15.html), or including audio description on videos. 
+
+
+### Provide text alternatives for images
+
+Alt attributes should describe the image they're applied to. For e.g. the alt attribute for a photograph should describe what's happening in the photograph, not just say "photograph". 
```

Comment:
`to. For e.g.` -> `to, e.g.` the "for" is part of the latin translation and so redundant there, i thought...